packages/component-library/scripts/icons/waterfall/README.md

```diff
@@ -0,0 +1,3 @@
+# Waterfall
+
```

Comment:
Most of this code is copied from elsewhere. Could we give credits to the original source by mentioning it in this readme?